gematria/datasets/pipelines/compile_modules_lib.py

```diff
@@ -109,6 +109,23 @@ def expand(self, pcoll: beam.PCollection) -> beam.PCollection:
 def get_bbs(
```

Comment:
Unrelated to this PR, and no action needed here. But we should see about turning this into a composite transform (per https://beam.apache.org/documentation/programming-guide/#composite-transforms).

Created #190 so that we don't forget about it.